### ISSUE TITLE
Company for ContactPerson billing

### ISSUE THREAD

**Issue by damikael** [italia/spid-sp-test#39]
https://github.com/italia/spid-sp-test/blob/d51dda2e4f5cfb3ed11f908ae9cdea18090b31bc/src/spid_sp_test/metadata_private.py#L28

See the example:
```
<md:ContactPerson contactType="billing">
     <md:Extensions xmlns:fpa="https://spid.gov.it/invoicing-extensions">
          <fpa:CessionarioCommittente>
               ...
          </fpa:CessionarioCommittente>
     </md:Extensions>
     <md:Company>Destinatario_Fatturazione</md:Company>
     <md:EmailAddress>email@fatturazione.it</md:EmailAddress>
     <md:TelephoneNumber>telefono_fatture</md:TelephoneNumber>
</md:ContactPerson>
```

**Comment by peppelinux** [comment]
Does these assertion are true?

1. CessionarioCommittente MUST be valued
2. Company MAY be valued and if present must be defined outside <Extensions>

**Comment by peppelinux** [comment]
fixed in v0.6.3
